net/kea/Makefile

```diff
@@ -1,6 +1,7 @@
 #
 # Copyright (C) 2019 Banglang Huang <banglang.huang@foxmail.com>
 # Copyright (C) 2019 Rosy Song <rosysong@rosinson.com>
+# Copyright (C) 2025 Liu Yu <f78fk@live.com>
```

Comment:
You want to be a new maintainer, right? That needs to be done in separate commit and if others are okay with it.

Reply:
Thanks for reviewing!
I don’t intend to become the new maintainer. I just want to share the upgrade to Kea 2.6.3 for others to use.
Please feel free to leave the maintainer as is.